pkg/cmd/server/start/start_master.go

```diff
@@ -498,14 +491,15 @@ func StartAPI(oc *origin.MasterConfig, kc *kubernetes.MasterConfig) error {
 		}
 
 		if oc.Options.AssetConfig.ServingInfo.BindAddress == oc.Options.ServingInfo.BindAddress {
-			unprotectedInstallers = append(unprotectedInstallers, config)
+			embeddedAssetConfig = config
+			proxyAssetConfig = config
```

Comment:
I attempted to trace the code to figure out which asset config we used when we were embedded vs proxying, and this appeared to be correct to me at the time